src/content/docs/en/reference/api-reference.mdx

```diff
@@ -778,10 +826,79 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 
 [See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
 
+### `getEntry()`
+
+<p>
+  <Since v="2.5.0" />
+</p>
+
+**Types:**
+
+- `(collection: string, contentSlugOrDataId: string) => CollectionEntry<collection>`
+- `({ collection: string, id: string }) => CollectionEntry<collection>`
+- `({ collection: string, slug: string }) => CollectionEntry<collection>`
+
+`getEntry()` is a function that retrieves a single collection entry by collection name and either the entry `id` (for JSON or YAML collections) or entry `slug` (for Markdown or MDX collections).
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+// Get `src/content/blog/enterprise.md`
+const enterprisePost = await getEntry('blog', 'enterprise');
+
+// Get `src/content/captains/captain-kirk.yaml`
+const captainProfile = await getEntry('captains', 'captain-kirk');
+---
+```
```

Comment:
```suggestion
```
I'm not sure I'd include this example at all, but rather continue right into getting the `data` etc., then if necessary, show all things in one sample. I think it's cleaner to state what this function can all be used for, then show the examples in one sample snippet.